plugins/devtools/src/alpha/plugin.tsx

```diff
@@ -46,14 +48,27 @@ export const devToolsApi = ApiBlueprint.make({
 });
 
 /** @alpha */
-export const devToolsPage = PageBlueprint.make({
-  params: {
-    path: '/devtools',
-    routeRef: convertLegacyRouteRef(rootRouteRef),
-    loader: () =>
-      import('../components/DevToolsPage').then(m =>
-        compatWrapper(<m.DevToolsPage />),
-      ),
+export const devToolsPage = PageBlueprint.makeWithOverrides({
+  inputs: {
+    routes: createExtensionInput([devToolsRouteDataRef], {
```

Comment:
```suggestion
    contents: createExtensionInput([
      coreExtensionData.reactElement,
      coreExtensionData.routePath,
      coreExtensionData.routeRef.optional(),
      // ....
    ], {
```

see [what the entity page does](https://github.com/backstage/backstage/blob/39a15fdf4dd1cc53da5dfa1563a69048e56d2d8d/plugins/catalog/src/alpha/pages.tsx#L79).

The reason that you need to leave these as dedicated outputs, is that this is how the routing system is able to detect that they exist and should be mapped to the routing table.

Reply:
I seeeee, now it also make sense why my plugin is not auto discovered